src/InsertionsClient/Api/DefaultConfigUpdater.cs

```diff
@@ -91,6 +91,7 @@ public bool TryLoad(string defaultConfigPath, out string error)
             }
             catch(Exception e)
```

Comment:
According to documentation,
> The loading functionality of LINQ to XML is built upon XmlReader. Therefore, you might catch any exceptions that are thrown by the **XmlReader.Create** overload methods and the XmlReader methods that read and parse the document.

XDocument.Load seems like it can throw many different exceptions. And the list of exceptions can change if the implementation of XmlReader changes. For the sake of stability, I think it is safer to just catch Exception for this case.